### DISCUSSION TITLE
Rotate animation will rebound at 0deg - 360deg or 360deg - 0deg

### DISCUSSION THREAD

**Discussion by ethanruiyu** [konvajs/konva#1235] [Q&A] [answered]
codesandbox: https://codesandbox.io/s/confident-star-fmbz1?file=/src/mobx.js

As you can see, It will rebound at 0deg - 360deg
My  source angle range is 0 -360

I want to achieve like this effect

https://user-images.githubusercontent.com/81229588/142389600-6a028d3d-4788-4f73-8089-4bf57c9ca106.mov

It bothered me for a long time 😭

**Comment by lavrton** [comment]
I am not certain if there are any simple solutions from Konva or `react-spring` for that. 

Why do you need to limit angle?

**Reply by ethanruiyu** [reply]
The original value is from the hardware sensor, which gives a range of 0-360, and the next value is random (because we can't predict sensor position and orientation).

**Reply by ethanruiyu** [reply]
Thanks for your answer. I have asked for help in `React-Spring` `Discussions`

**Comment by raycaster08** [comment, marked as answer]
Modify the original value based on the difference
https://codesandbox.io/s/flamboyant-ioana-5zems1?file=/src/App.js

**Reply by ethanruiyu** [reply]
nice fix
